typings/glamorous.d.ts

```diff
@@ -40,6 +40,12 @@ interface ThemeProps {
 
 export class ThemeProvider extends React.Component<ThemeProps, any> { }
 
+export declare function withTheme
+  <P>(
+    component:
+      | React.ComponentClass<P>
```

Comment:
The method of testing you're using there is completely opaque to me and I'm not sure what a valid test is. And again, I'm completely TypeScript-illiterate. So I've taken a guess but TS isn't happy with it and I can't stumble on something that works... Here's what I tried:

```js
import glamorous, { withTheme, ThemeProvider } from "../";
// ...
const ThemedClass = withTheme(ClassToWrap)({})
```

But I get: "Value of type 'ComponentClass<object & ExtraGlamorousProps>' is not callable. Did you mean to include 'new'?"

Suggestions?